### ISSUE TITLE
Filter out archive files from DMM result list

### ISSUE THREAD

**Issue by eyalgindi** [iPromKnight/zilean#28]
Using Prowlarr to bridge Zileant and RDTClinet (symlinks mode) and occasionally which works great for most files.
Yet, qBt, fails when it comes to compressed files.

`Cant handle compressed files with symlink downloader for download`

Wanted to set DEBUG logging level and explore the results struct Zilean returns to Prowlarr and filter those on Prowlarr side, but couldn't find a way to control log verbosity (is there?).

 Bottom line, would be great if Zilean could expose a configurable way to control unwanted file types\extensions.

**Comment by eyalgindi** [comment]
After further investigation.
Apparently, creating an archive is the default behaviour of the RD service whenever the download client asks for more than a single file. Archiving can be avoided by limiting qBt to download a single file only, by limiting the min file size to be larger of a desired threshold so not small files (samples, subs, metadata, etc) would be requested.  

Closing the issue, thank you.
